src/appleseed/renderer/modeling/project-builtin/cornellboxproject.cpp

```diff
@@ -1014,6 +1017,43 @@ auto_release_ptr<Project> CornellBoxProjectFactory::create()
         // Attach the camera to the scene.
         scene->cameras().insert(camera);
     }
+    
+    //
+    // Environment
+    //
+
+    // Color called "black_radiance" is created and inserted into the scene.
+    static const float black_radiance[] = { 0.0f, 0.0f, 0.0f };
+    scene->colors().insert(
+        ColorEntityFactory::create(
+            "black_radiance",
+            ParamArray()
+                .insert("color_space", "srgb")
+                .insert("multiplier", "0.5"),
```

Comment:
Why 0.5?

Reply:
@dictoon; I am actually a bit confused about what multiplier is being referred to as here. Can you please elaborate what is it and what it's value should be in this case?

Reply:
You can take a look at the metadata of ColorEntityFactory and see what is the default value for "multiplier". Maybe this can be a good value.

Reply:
That's a multiplier for the color components. Since the components of this color are all 0.0 the multiplier makes no difference, but I would prefer not to have an arbitrary 0.5 multiplier. Since this parameter is optional, I would simply recommend to remove it.

Reply:
done